### ISSUE TITLE
Several things not working

### ISSUE THREAD

**Issue by Kahz3l** [manualdousuario/sintoniza#1]
Hi Manualdousario,

I tested your application and several things are wrong in the documentation:
```
      DB_USERNAME -> DB_USER
      DB_PASSWORD -> DB_PASS
```

When I fixed all this, sintoniza started, but clicking on Login, Register or Statistics did nothing.
In logs there is just:
`127.0.0.1 - 14/Nov/2024:16:11:44 +0100 "GET /index.php" 200
`
I tried Mariadb 11.4, 10.11, both do not work. I also tried sintoniza 1.10.7 instead of latest 1.10.8, also did not work.

![image](https://github.com/user-attachments/assets/ae1c2553-fe95-4f4d-8022-cbec21dbb8b0)
See URL at the top.
The instance on your public instance does work though so I don't know where the error is.

**Comment by altendorfme** [comment]
Hello,

You're right, I just fixed the documentation and compose.

About accessing internal pages, confirm if the `BASE_URL` has a `/` at the end, for example `BASE_URL: https://sintoniza.xyz/`
This is a small bug that was inherited from oPodSync and is in the queue to be fixed.

**Comment by Kahz3l** [comment]
Thanks for the tip with the slash, this was the problem!

I have another problem now, that also occurred on your public instance. 
When updating all feed metadata it will throw an internal error:
![Screenshot_20241114_225414_Brave](https://github.com/user-attachments/assets/ae97296c-ae86-4008-b887-a051bb86bcec)
Logs:
```
127.0.0.1 -  14/Nov/2024:22:53:28 +0100 "GET /index.php" 500

NOTICE: PHP message: TypeError: format_description(): Argument #1 ($str) must be of type string, null given, called in /app/templates/dashboard/subscriptions.php on line 108 and defined in /app/index.php:39

Stack trace:

#0 /app/templates/dashboard/subscriptions.php(108): format_description(NULL)

#1 /app/index.php(89): require_once('/app/templates/...')

#2 {main}

NOTICE: PHP message: TypeError: format_description(): Argument #1 ($str) must be of type string, null given, called in /app/templates/dashboard/subscriptions.php on line 108 and defined in /app/index.php:39

Stack trace:

#0 /app/templates/dashboard/subscriptions.php(108): format_description(NULL)

#1 /app/index.php(89): require_once('/app/templates/...')

#2 {main}

NOTICE: PHP message: PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'media_url' cannot be null in /app/inc/DB.php:144

Stack trace:

#0 /app/inc/DB.php(144): PDOStatement->execute()

#1 /app/inc/DB.php(111): DB->simple('INSERT INTO epi...', Array)

#2 /app/inc/Feed.php(53): DB->upsert('episodes', Array, Array)

#3 /app/inc/GPodder.php(299): Feed->sync(Object(DB))

#4 /app/inc/GPodder.php(348): GPodder->updateFeedForSubscription(5)

#5 /app/templates/dashboard/subscriptions.php(7): GPodder->updateAllFeeds()

#6 /app/index.php(89): require_once('/app/templates/...')

#7 {main}

NOTICE: PHP message: PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'media_url' cannot be null in /app/inc/DB.php:144

Stack trace:

#0 /app/inc/DB.php(144): PDOStatement->execute()

#1 /app/inc/DB.php(111): DB->simple('INSERT INTO epi...', Array)

#2 /app/inc/Feed.php(53): DB->upsert('episodes', Array, Array)

#3 /app/inc/GPodder.php(299): Feed->sync(Object(DB))

#4 /app/inc/GPodder.php(348): GPodder->updateFeedForSubscription(5)

#5 /app/templates/dashboard/subscriptions.php(7): GPodder->updateAllFeeds()

#6 /app/index.php(89): require_once('/app/templates/...')

#7 {main}

127.0.0.1 -  14/Nov/2024:22:53:30 +0100 "POST /index.php" 500
```
Maybe there's an error?

**Comment by altendorfme** [comment]
Fixed in version 1.11.2 and updated in public instance

I will temporarily limit the update of feed meta data to cron, to avoid timeouts. This is another improvement I need to make, the asynchronous update of meta data.

**Comment by Kahz3l** [comment]
Thank you very much for the improvements, it looks great now, good job!
I will try Kasts on Windows later. 
AntennaPod 3.5.0 (300baf6dd) on android 14 works with it.
